src/ISA/ISA/Regex.fs

```diff
@@ -88,11 +213,11 @@ open System.Text.RegularExpressions
 /// 
 /// **Example 2:** "Term Source REF (NFDI4PSO:0000064)" --> "NFDI4PSO:0000064"
 /// </summary>
-let tryParseTermAccession (str:string) =
+let tryParseTermAnnotation (str:string) =
```

Comment:
Not sure why you renamed this function and all related patterns from `Accession` to `Annotation`, as the information always comes from "Term Accession Number" column in ISA-TAB

Reply:
The information either comes from `Term Accession Number` or `Term Source REF`. And what you get as a result is accession number `AND` term source ref.